includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -23,12 +23,24 @@ public function setHrefs( $hrefs ) {
 	/**
 	 * @param $name string key for href
 	 * @param $lang string two letter language code
+	 * @param [$variables] array key-value variables that will be replaced in the href
 	 * @return string full URL, in case of lang specific URL missing, default one is returned
 	 */
-	public function getHref( $name, $lang ) {
+	public function getHref( $name, $lang, $variables = null ) {
```

Comment:
Do we really need to make this method more complex? It's the first use-case of URL that is based on variable and here concatenation is enough.
If we really want to keep this functionality, please cover it with unit tests.